Template/Tag/Etracker.svg

```diff
@@ -0,0 +1,17 @@
+<svg width="56" height="53" viewBox="0 0 56 52" xmlns="http://www.w3.org/2000/svg">
+<!-- Source: PUT_SOURCE_HERE -->
```

Comment:
Really minor nitpick, but I meant that you should put the source of the file here, so we credit the author :slightly_smiling_face: